Relativity.Shared.SDK.md

```diff
@@ -14,4 +20,4 @@ Initial Version
 
 Lowest Version | Highest Version
 --- | ---
-11.3.185.7 | Latest
\ No newline at end of file
+11.3.185.7 | Latest

```

Comment:
It might seem like overkill, but we ask that users copy the table (and update it if necessary) for each row of the changelog)
You can check out this one to see how it looks: 
https://github.com/relativitydev/relativity-api-documentation/blob/master/Relativity.Audit.SDK.md

Reply:
No worries. Done